test/test_model_regression.py

```diff
@@ -45,13 +54,25 @@ def md5(fname):
       hash_md5.update(chunk)
   return hash_md5.hexdigest()
 
-# set global tolerances for floating point comparisons (see np.testing.assert_allclose)
+# set global tolerances for floating point comparisons. 
+# From documentation on np.testing.assert_allclose
+# the comparison is
+#
+#     absolute(`a` - `b`) <= (`atol` + `rtol` * absolute(`b`))
+#
+# Setting atol=0 means we are only testing relative errors
+#
 atol = 0.0
 # why a high tolerance? For some reason, a high tolerance is needed when 
 # comparing to regression data on different machines
-# TODO: explore the origin of these large discrepancies (note that hdf5 data is saved in single precision, and errors seem to be post-merger)
-rtol_gsl = 0.75
-rtol = 1.e-11
+# TODO: explore the origin of these large discrepancies 
+#       note that regression data model_regression_data.h5 is saved in single precision (see above)
+#       largest relative errors seem to be post-merger
+#       only seems to affect models that use gpr fits and/or gsl calls
+rtol                   = 1.e-11
```

Comment:
I thought we were switching to atol instead?

Reply:
The issue with rtol was mostly due to using h+ and hx for the regression test, and these pass through 0 which led to jumps in the relative error. Switching the comparison to use h+ + ihx fixed that problem. There's still a second potential issue that the amplitude becomes very small in the ringdown (which on last Tuesday I thought was the primary problem) ... but this turned out to be a non-issue as the relative errors only increase by about a factor of 5 (for the GPR-based models) in the deep ringdown.